_scripts/deploy_key_decrypt.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash 
-# Decrypt Github Deploy key for the Semantic Release package to be able to push tags 
-# Ubuntu-14.04
-
-# Make sure only the current user can read the private key- 
```

Comment:
Can we add back all of these comments to the new script too?

Reply:
Yep, done.

Reply:
Thanks